### DISCUSSION TITLE
URL prefill does not seem to work for `checkboxes` or `dropdown`

### DISCUSSION THREAD

**Discussion by anomiex** [community/community#5288] [Projects and Issues]
At https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/syntax-for-githubs-form-schema#keys, it says of the `id` attribute (emphasis added)

> The identifier for the element, except when `type` is set to `markdown`. Can only use alpha-numeric characters, `-`, and `_`. Must be unique in the form definition. **If provided, the `id` is the canonical identifier for the field in URL query parameter prefills.**

Specifying the `id` in a URL works for `input` and `textarea` types, but I can't figure out a way to get it to work for `checkboxes` or `dropdown`.

**Comment by lars-reimann** [comment]
It would be great if it would be documented how to make prefilling work with checkboxes and dropdowns or implement this feature in the first place. I also haven't been able to figure out how this works. The easier and faster users can provide quality feedback, the better. 

I also haven't been able to figure out how this works. I've tried setting the URL parameters to the text of the corresponding option as well as using indexes.

Just switching to text areas for everything is unfortunately not an option, since users should also be able to use the issue forms without prefilling directly from GitHub.

**Reply by Craftplacer** [reply]
I have also tried assigning keys to the options (example down below) but GitHub doesn't seem to like that, although this would seem like the best solution.

```yaml
options:
  firstOption: "Option 1"
  secondOption: "Option 2"
  thirdOption: "Option 3"
```

**Comment by matypist** [comment]
Would be very easy to support and very useful at the sametime, :up:
